googletest/src/gtest-port.cc

```diff
@@ -58,6 +58,10 @@
 # include <sys/procfs.h>
 #endif  // GTEST_OS_QNX
 
+#if GTEST_OS_AIX
+# include <procinfo.h>
```

Comment:
Please add it explicitly. I don't want to rely on the presence of other includes. Those could disappear.